sponsor-dapp-v2/src/components/Step3.js

```diff
@@ -45,6 +45,7 @@ function useCreateContract(userSelectionsRef, identifierConfig, onNextStep, stat
       userSelectionsRef.current.contractAddress =
         TXObjects[TXObjects.length - 1].receipt.events.CreatedTokenizedDerivative.returnValues.contractAddress;
 
+      userSelectionsRef.current.symbol = state.tokenSymbol;
```

Comment:
Why was this necessary?

Reply:
As discussed IRL, seems like the easiest way to move the symbol on to later stages.